src/blog/2022/011-form-js-0.10.0.md

```diff
@@ -0,0 +1,141 @@
+---
+
+title: Conditional Rendering, New Components & Expression Properties in form-js
+description: "Introducing form-js@0.10.0: This release introduces conditional rendering, new form components & FEEL expression properties."
+preview_image: assets/attachments/blog/2022/003-preview.png
```

Comment:
Should we update this preview? Pick a reasonable shot to be showcased in social media, when linking to this post.

Reply:
Good catch! Missed that one